codes/cin_cout.cpp

```diff
@@ -2,11 +2,13 @@
 // cin , cout
 
 #include <iostream>
+#include <string>
 using namespace std;
 
 int main() {
     int A;
     cin >> A;
-    cout << A*A;

```

Comment:
ehsas mikonam inja gharar bood error mikhord
inke biyaym bara computer yechizi ro tarif konim bad begim ye dastoor ro bede biroon gheyr manteghi omad
mizanam compiler bebinam kar mikone ya na

Reply:
check  it and tell me.